npm_and_yarn/spec/fixtures/projects/npm7/simple/package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "{{ name }}",
+    "name": "project-name",
```

Comment:
This is more realistic example where the lockfile has a "valid" generated name by some build step, we make sure to restore this name, not the invalid name in the `package.json`, the code to restore this hasn't changed